roles/build/tasks/main.yml

```diff
@@ -134,4 +134,13 @@
     shell:
       cmd: "source /home/vagrant/.bashrc && nvm use 12 && npm install --global yarn"
 
+
+  - name: "Install node version 14"
+    shell:
+      cmd: "source /home/vagrant/.bashrc && nvm install 14"
```

Comment:
Loading the environment varables before calling the `nvm` works fine I guess. Alternatively consider running `nvm` from out of its installation folder. `/usr/local/bin/nvm install 14`. Or perhaps install NVM into a bin folder that is available in the environment.

Reply:
nvm used here is implemented as a shell function that is initialized from .bashrc by sourcing .nvm/nvm.sh. Because each Ansible command runs in a new shell it needs to be initialized each time, and because it modifies the shell environment it must be sourced. That is why I did it like this. There is no nvm executable. Maybe there is another method of using nvm?

Reply:
I think you already dove deeper than I did @pmeulen. As an alternative to `nvm`, there is another very popular `npm` package named `n`. I see no great benefit to switch to using `n`. No need to fix if it ain't broken.

`https://github.com/tj/n`